src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -1089,6 +1116,14 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
         return "";
     }
 
+    @computed get alterationTypesInQuery() {
```

Comment:
I know we've talked about this before but I really prefer @memoized and then you pass in the dereferenced result.  That we you deal with the completion status outside

Reply:
you mean make a function wrapping line 1121?